lib/user-interface/react/src/components/chatbot/Message.tsx

```diff
@@ -100,8 +94,8 @@ export default function Message ({ message, isRunning, showMetadata, isStreaming
                         <div style={{ maxWidth: '60em' }}>
                             {markdownDisplay ? <ReactMarkdown
                                 remarkPlugins={[remarkBreaks]}
-                                children={getDisplayableMessage(message.content, message.metadata)}
-                            /> : <div style={{ whiteSpace: 'pre-line' }}>{getDisplayableMessage(message.content, message.metadata)}</div>}
+                                children={getDisplayableMessage(message.content, !isStreaming && message.metadata?.ragDocuments ? message.metadata.ragDocuments : undefined)}
+                            /> : <div style={{ whiteSpace: 'pre-line' }}>{getDisplayableMessage(message.content, !isStreaming && message.metadata?.ragDocuments ? message.metadata.ragDocuments : undefined)}</div>}
```

Comment:
`!isStreaming && message.metadata?.ragDocuments ? message.metadata.ragDocuments : undefined` is repeated and would be a lot more readable broken out